### DISCUSSION TITLE
Shared documents expiration and permission

### DISCUSSION THREAD

**Discussion by alissa-tung** [toeverything/AFFiNE#6256] [Ideas]
### Description

- a view, containing all pages have been shared, and their links
- who make the page public
- expire a public link
- how many external people had access the page
- ......

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by lawvs** [comment]
> a view, containing all pages have been shared, and their links

For the first question, you can create a dynamic collection to filter the shared page

<img width="333" alt="Screenshot 2024-03-21 at 15 51 52" src="https://github.com/toeverything/AFFiNE/assets/18554747/32140e38-6e20-440d-943c-19cc7939eaba">
